### PR TITLE
fix: the bug when pasting the data in IE11(markdown)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed the bug when pasting the data in IE11(markdown)
  * removed blank line and parsing error

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
리뷰 완료입니다~

File: apps/editor/src/helper/image.ts [hunk 1]
```diff
@@ -12,21 +12,12 @@ export function addDefaultImageBlobHook(eventEmitter: Emitter) {
...
-    eventEmitter.emit(
-      'command',
-      { type: editorType, command: 'addImage' },
-      {
-        imageUrl,
-        altText: altText || blob.name || 'image',
-      }
-    );
+    eventEmitter.emit('command', 'addImage', {
+      imageUrl,
```
Comment (jungeun-cho) [inline apps/editor/src/helper/image.ts:18]: `editorType`이 없어지고 hook 리스너가 등록되어있는 모듈(?)에 알아서 전달되도록 변경된 것인가요?
#1515 에서 `setCodeBlockLanguages` => `addHook`작업과 관련있는 것인지요...?

**Review by jungeun-cho — APPROVED** [review]
확인했습니다.
작업하신 내용중에 궁금한 내용 코멘트 남겨놓았습니다~!ㅎㅎ

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/editor/src/helper/image.ts (hunk 1 above)
Comment (js87zz) [reply apps/editor/src/helper/image.ts:18]: #1451 에서 커맨드 API관련 구조가 변경되었는데 이 부분에 적용되지 반영하였습니다!

**Pull request merged**